<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSignalDialog</class>
 <widget class="QDialog" name="AddSignalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>479</width>
    <height>468</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddSignalDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>常规</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>*描述：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_miaoshu</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_miaoshu">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>系数：</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_xishu</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_xishu">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.001</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>单位：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_unit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="2">
       <widget class="QLineEdit" name="lineEdit_unit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>*开始地址(D)：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_addr</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_addr">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="4" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>寄存器：</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_reg</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QComboBox" name="comboBox_reg">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>22</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>数据类型</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>数据类型：</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_datatype</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_datatype">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>short</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>unshort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>int(abcd)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>int(cdab)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>int(badc)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>int(dcba)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>unint(abcd)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>unint(cdab)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>unint(badc)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>unint(dcba)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float(abcd)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float(cdab)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float(badc)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float(dcba)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>string</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>偏移量：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_offset</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_offset">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>范围</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>最小值：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_min</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_min">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>最大值：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_max</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_max">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>信号枚举显示</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="3">
       <widget class="QTableWidget" name="tableWidget_enum">
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Display</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_add">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_del">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>69</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
